tests/foreman/cli/test_discoveryrule.py

```diff
@@ -245,9 +245,16 @@ def test_positive_create_with_priority(self):
 
         :CaseImportance: Critical
         """
-        rule_priority = '100'
+        rule_list = DiscoveryRule.list()
+        p_list = []
+        for rule in rule_list:
+            p_list.append(rule['priority'])
+        while True:
+            rule_priority = gen_integer(1, 100)
```

Comment:
why limiting this to just 100? i'd increase this to at least 1000 to lower the probability of a conflict

Reply:
your right, i will change it as well

Reply:
pls, check my suggestions in the review comments